Gemfile

```diff
@@ -1,18 +1,10 @@
-source "http://rubygems.org"
+# A sample Gemfile
+source "https://rubygems.org"
 
-if ENV.key?('PUPPET_VERSION')
```

Comment:
Why drop the ability to test against multiple versions of Puppet? We tend to see a number of versions in the wild and it's useful to be able to test against a set of known good versions as well as the latest.